src/components/divider/index.jsx

```diff
@@ -0,0 +1,27 @@
+import PropTypes from "prop-types";
+
+const COMPONENT_CLASS_NAME = "c-divider";
+
+const Divider = ({ className, assistiveHidden }) => {
+	let ariaProp = {};
```

Comment:
```suggestion
	const ariaProp = {};
```

Reply:
Fixed it in [2c036af](https://github.com/WPMedia/arc-themes-components/pull/145/commits/2c036afe131097f7f3991c2ae7546efc2b527f7c)